eagle-core/eagle-metadata/eagle-metadata-jdbc/src/main/java/org/apache/eagle/metadata/store/jdbc/service/ApplicationEntityServiceJDBCImpl.java

```diff
@@ -116,6 +116,14 @@ public ApplicationEntity delete(ApplicationEntity applicationEntity) {
         return entity;
     }
 
+    /**
+     * TODO: UPDATE ApplicationEntity through JDBC is not supported yet
+     */
+    @Override
+    public ApplicationEntity update(ApplicationEntity entity) {
+        throw new UnsupportedOperationException("UPDATE ApplicationEntity through JDBC is not supported yet");
```

Comment:
@r7raul1984 Could you please implement updating operation for JDBC metadata store?